packages/docs/src/components/Layout.tsx

```diff
@@ -67,9 +61,11 @@ const Layout = ({
       </Helmet>
 
       <SkipNav href="#main" />
-      <Header />
+
+      <UsaBanner className="ds-u-display--none ds-u-md-display--block" />
+
       <div className="ds-l-row ds-u-margin--0">
-        <Sidebar isMobileNavOpen={isMobileNavOpen} location={location} />
+        <Navigation location={location} />
```

Comment:
In the mural, it looks like the nav extends all the way to the bottom of the page as its own column instead of stopping at the footer like it does now. Is that something that should be included in this PR or was that part of a future ticket?
<img width="599" alt="Screen Shot 2022-06-23 at 4 02 21 PM" src="https://user-images.githubusercontent.com/33579665/175422945-978cb5fa-9c9c-4e63-8016-7c3ad4696566.png">

Reply:
I'm open to interpretation here - it might be nice for the footer to go full width. This will then align with the homepage design Melissa did.